great_expectations/data_context/data_context/abstract_data_context.py

```diff
@@ -1608,6 +1660,70 @@ def add_store(self, store_name: str, store_config: StoreConfigTypedDict) -> Stor
         self._save_project_config()
         return store
 
+    @public_api
+    @new_method_or_class(version="0.17.2")
+    def add_data_docs_site(
```

Comment:
Should this error if an existing resource is around?

Reply:
Good question - I was aiming for consistency - let me check a few other add methods. Is there a canonical doc on what our CRUD should do?

Reply:
It looks like in:
- datasource we overwrite
- checkpoint we overwrite but that is deprecated in favor of raising in `add_checkpoint` and upserting in `add_or_update_checkpoint`
- add_expectation_suite uses a flag to choose between `add` and `add_or_update` on the underlying store.

So it seems like we should error if a site exists but let me know if I'm mistaken!

Reply:
I think so! Does feel a little inconsistent but I prefer erroring